UPGRADE-7.1.md

```diff
@@ -69,6 +120,7 @@ Form
 FrameworkBundle
 ---------------
 
+ * [BC BREAK] Enabling `framework.rate_limiter` requires `symfony/rate-limiter` 7.1 or higher
```

Comment:
Does FrameworkBundle 7.1 conflict with rate-limiter < 7.1 ? If yes, there is no BC break as composer deals with ensuring compatible versions.

Reply:
No, it doesn't. It's only part of the `require-dev` of framework bundle.